src/ngResource/resource.js

```diff
@@ -702,12 +702,13 @@ angular.module('ngResource', ['ng']).
               return $q.reject(response);
             });
 
-            promise.finally(function() {
+            promise = promise.finally(function(response) {
```

Comment:
That all makes sense... but `response` is undefined 100% of the time because the `.finally` callback doesn't take any arguments. You can drop the `response` arg and return statement...